refresh.template.py

```diff
@@ -683,7 +694,36 @@ def _get_apple_active_clang():
     # Unless xcode-select has been invoked (like for a beta) we'd expect, e.g., '/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang' or '/Library/Developer/CommandLineTools/usr/bin/clang'.
 
 
-def _apple_platform_patch(compile_args: typing.List[str]):
+@functools.lru_cache(maxsize=None)
+def _get_apple_active_swiftc():
+    """Get path to xcode-select'd swiftc version."""
+    return subprocess.check_output(
+        ('xcrun', '--find', 'swiftc'),
+        stderr=subprocess.DEVNULL, # Suppress superfluous error messages like "Requested but did not find extension point with identifier..."
```

Comment:
It looks to me from a quick test like this has been fixed in the latest Xcode.

That is, `xcrun --find clang --no-cache` and `xcrun --find swiftc --no-cache` now run clean--and I really shouldn't be blanket ignoring error messages needlessly. Would you mind removing stderr>devnull here and for clang? (As a favor. My issue, but good to keep the two in sync.)

Reply:
ok

Reply:
https://github.com/hedronvision/bazel-compile-commands-extractor/commit/f9ceaf3b7bf5579d7b1793571e5dafe35e4621e2

Reply:
Ah, sorry. The `--no-cache` was just for testing--and would have exposed the superfluous error bug if Apple hadn't fixed it.  Thanks for fixing! (And, I'm realizing, giving me commit access. I should have just done.)